letsencrypt-auto-source/Dockerfile.redhat6

```diff
@@ -35,9 +39,10 @@ RUN curl https://raw.githubusercontent.com/certbot/certbot/v0.38.0/letsencrypt-a
     -o /home/lea/certbot/letsencrypt-auto-source/letsencrypt-auto_py_34 \
  && chmod +x /home/lea/certbot/letsencrypt-auto-source/letsencrypt-auto_py_34
 
-RUN chmod +x /home/lea/certbot/letsencrypt-auto-source/tests/centos6_tests.sh
+RUN cp /home/lea/certbot/letsencrypt-auto-source/tests/${REDHAT_DIST_FLAVOR/\/sl/}6_tests.sh /home/lea/certbot/letsencrypt-auto-source/tests/redhat6_tests.sh \
```

Comment:
It is an in-place string substitution, that executes a substitution on the variable value, then interpolate the result as the output string. Format is "${VAR/old/new}", which outputs the `VAR` value with occurrences of `old` replaced by `new`.

I use it to remove `/sl` from `REDHAT_DIST_FLAVOR` (so empty string after the `/`) in the case of scientific linux 6, since the associated docker image is `scientificlinux/sl`, and I do not appreciate to have a `/` in the name of a file.

Reply:
Oh I see! I never knew you could do that.

Can we either:

1. Remove it since we're not testing on Scientific Linux and I doubt we ever will.
2. Add a comment explaining why we're doing this mentioning the name of the Scientific Linux image.

Reply:
Ah yes, you are right, it is not useful anymore. I remove it.